src/nodes/VariableDeclaration.js

```diff
@@ -29,6 +31,7 @@ const VariableDeclaration = {
           visibility(node),
           constantKeyword(node),
           storageLocation(node),
+          immutable(node),
```

Comment:
qq: is this the right position for the immutable keyword btw? // cc @fvictorio @Janther

Reply:
I think it's OK. An immutable variable cannot also be constant, and storage location doesn't make sense for state variables (which are the only ones that can be immutable), so for all effects this is just putting it after the visibility node, which makes sense.